### ISSUE TITLE
ChildIOException: File/directory is a child to another output:

### ISSUE THREAD

**Issue by lindenb** [aryarm/varCA#23]
Hi,
I'm trying to run the example dataset.  (snakemake `5.19.2` ) My server crashed during the workflow and I tried to re-run it.

```
$ ./run.bash --sge-cluster && echo OK
$ 
```

this the the content of the log file:
```
Building DAG of jobs...
ChildIOException:
File/directory is a child to another output:
('/SCRATCH-BIRD/users/lindenbaum-p/work/20210312.varCA/merged_snp/jurkat_chr1/norm.tsv.gz', result)
('/SCRATCH-BIRD/users/lindenbaum-p/work/20210312.varCA/merged_snp/jurkat_chr1/norm.tsv.gz', norm_nums)
```

can you help me please ?

**Comment by aryarm** [comment]
Hi @lindenb,
Thanks for trying out VarCA!
Can you help me reproduce the issue? Do you know, by any chance, at which stage in the pipeline your server crashed?
While the pipeline should theoretically work with Snakemake 5.19.2, it was officially built on 5.18.0, so I will try to reproduce the issue with the version of Snakemake that you're using.

**Comment by aryarm** [comment]
Just providing an update on this now that I've had some time to think on it and do some googling.

I now suspect that this was due to a bug in Snakemake, since I hadn't seen this in v5.18.0. According to [this GitHub issue](https://github.com/snakemake/snakemake/issues/413), the problem arose somewhere around v5.19.0 and [was patched in v5.32.0](https://github.com/snakemake/snakemake/pull/825).
I'm going to close this issue for now, but feel free to reopen if anyone runs into this problem with a version of Snakemake above v5.32.0. In any case, I soon plan to migrate the pipeline to a version of Snakemake > v6.0.0, and this issue should be irrelevant at that point.
